solvers/Dummyclassifier.py

```diff
@@ -0,0 +1,63 @@
+from benchopt import BaseSolver, safe_import_context
```

Comment:
why is this for useful?

Reply:
Moved Dummy Classifier to Moabb Pipelines